client/a8c-for-agencies/sections/migrations/consolidated-commissions/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { Card } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+
+import './style.scss';
+
+const getCurrentQuarter = () => {
+	const currentMonth = new Date().getMonth();
+	const quarter = Math.ceil( ( currentMonth + 1 ) / 3 );
+	return `Q${ quarter }`;
+};
+export default function MigrationsConsolidatedCommissions() {
```

Comment:
Nitpick: Let's insert an empty after the function declaration.

Reply:
Fixed it: https://github.com/Automattic/wp-calypso/pull/96225/commits/fbed8c0b6f4dbb1ac7dfb25e60a5dc6748342779